eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -93,6 +93,15 @@ jobs:
       vmImage: '$(OSVmImage)'
 
     steps:
+    - pwsh: |
+        $setDailyDevBuild = "false"
+        if (('$(Build.Reason)' -eq 'Schedule') -and ('$(System.TeamProject)' -eq 'internal')) {
+          $setDailyDevBuild = "true"
+        }
+        echo "##vso[task.setvariable variable=SetDevVersion]$setDailyDevBuild"
```

Comment:
Nit: `SetDevVersion` sounds more like a function name than a variable name to me. Something like `IsDevBuild` or the like jumps out at me as a boolean variable.

Reply:
This is done to keep it consistent with other language repo and as per suggestion from Wes.